hugging_py_face/computer_vision.py

```diff
@@ -45,4 +48,21 @@ def object_detection(self, inputs: Union[Text, List], model: Optional[Text] = No
         if type(inputs) == list:
             return self._query(inputs, model=model, task="object-detection")
         elif type(inputs) == str:
-            return self._query(inputs, model=model, task="object-detection")
\ No newline at end of file
+            return self._query(inputs, model=model, task="object-detection")
+        
+    def object_detection_in_df(self, df: DataFrame, column: Text, model: Optional[Text] = None) -> DataFrame:
+        """
```

Comment:
Something else that I was thinking about is, given the situation that multiple objects are detected in a single image, we will only be returning the first one. Maybe we can include all of the objects detected for a particular image in separate rows? What do you feel about this?

Reply:
Yeah I noticed this behavior. I think having all objects might be a good option and allows for extensibility. In my case I was happy with a single result for the first iteration. Having any accurate result was a huge win to start as far as I was concerned.

Reply:
Awesome! Let's see what we can do to support this.